README.md

```diff
@@ -1,7 +1,16 @@
 # quantum-attendance
 An attendance program for any type of martial arts dojo.
 
-# Set up for local development
+# Set up for local development (with Docker)
+
+ 1. Install Docker
+ 2. run `docker-compose up`
+
+The first run might take a while due to dependencies being installed.
+In case dependencies change and the docker image needs to be rebuilt, just use 
+`docker-compose up --build`.
```

Comment:
Keeping the intermediate containers is exactly what we want! Otherwise the environment will be set up from scratch every time you run `docker-compose up` - setting up the DB, installing dependencies, etc.